src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -47,6 +53,21 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\AES\AesFactory.cs">
       <Link>CommonTest\AlgorithmImplementations\AES\AesFactory.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\DSA\DSAFactory.cs" Condition="'$(TargetGroup)'=='netstandard1.7'">
```

Comment:
@joperezr for example, for recent work in Algorithms I added several new files (e.g. DSA.cs) but didn't specify that it was for netstandard1.7 only. Thus they will be included in netstandard1.6. Is this desired? In this example it is a new file, but in other cases, it can get more complex as new APIs can be added in an existing file, and intertwined with existing APIs (e.g. an older API may call a new API) so simple ifdefing them out wouldn't work.